src/AspNet.Security.OAuth.Docusign/DocusignAuthenticationOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.Docusign;
+
+/// <summary>
+/// Defines a set of options used by <see cref="DocusignAuthenticationHandler"/>.
+/// </summary>
+public class DocusignAuthenticationOptions : OAuthOptions
+{
+    /// <summary>
+    /// Sets or retrieves a value that determines whether development or production endpoints are used.
+    /// The default value of this property is <see cref="DocusignAuthenticationEnvironment.Development"/>.
```

Comment:
This feels like a foot-gun to me - I would have thought production being the default would be less-surprising/safer.